azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -641,7 +641,14 @@ private void getRunningFlowsIdsHelper(List<Integer> allIds,
 
   @Override
   public List<ExecutableFlow> getRecentlyFinishedFlows() {
-    return new ArrayList<ExecutableFlow>(recentlyFinished.values());
+    List<ExecutableFlow> flows = new ArrayList<>();
+    try {
+      flows = executorLoader.fetchRecentlyFinishedFlows(
+          RECENTLY_FINISHED_LIFETIME.toMillis());
+    } catch(ExecutorManagerException e) {
+      logger.error(e);
```

Comment:
This will call
void error(Object message) {
which will not log the exception details such as call stack.
Always use this method to log an exception. 
void error(Object message, Throwable t) {